app/src/main/java/com/infomaniak/swisstransfer/ui/components/transfer/TransferItemList.kt

```diff
@@ -63,11 +67,17 @@ fun TransferItemList(
             contentType = { getTransfers()[it] },
             itemContent = {
                 val transfer = getTransfers()[it]
-                TransferItem(
-                    transfer = transfer,
-                    isSelected = { selectedTransferUuid == transfer.uuid },
-                    onClick = { onClick(transfer) },
-                )
+                SwipeToDismissComponent(
```

Comment:
Is it better to directly call `CustomShapes.SMALL` instead of creating another `val` ? It's not costly since `CustomShapes.SMALL` is a constant and we don't have to look way on the top to know what is `itemShape`.

Reply:
The shape is used twice, and it needs to be the same.